views/Todo/Todo.js

```diff
@@ -1,17 +1,63 @@
 import React from "react";
-import { View } from "react-native";
+import { Text, View } from "react-native";
 import Error from "../../components/Error/Error";
 import Header from "../../components/Header/Header";
 import Loading from "../../components/Loading/Loading";
 import TextArea from "../../components/TextArea/TextArea";
 
 export default function Todo({ loading, error, data, insets }) {
```

Comment:
booleans should be named with a verb state so that they are clearly read as booleans. isLoading, hasError